modular_skyrat/modules/bluespace_miner/code/bluespace_miner.dm

```diff
@@ -135,7 +135,7 @@
 /datum/supply_pack/misc/bluespace_miner
 	name = "Bluespace Miner"
 	desc = "Nanotrasen has revolutionized the procuring of materials with bluespace-- featuring the Bluespace Miner!"
-	cost = CARGO_CRATE_VALUE * 1000 // 200,000, one of the most expensive buys
+	cost = CARGO_CRATE_VALUE * 375 // 75,000
```

Comment:
i have warned you about the effects this will have on resource scarcity, i really don't think the impact that this will have on the system is good for the mining job or for the gameplay of cargo at all